tests/cases/fourslash/getOutliningForArrowFunctionBody3.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="fourslash.ts"/>
+
+////declare function foo(props: any);
+////foo(
+////    (a, b, c) =>[| {
```

Comment:
Do we already have tests for the multiline parameter list case, like

```ts
(a,
 b,
 c) => { }
```

Reply:
And I'm not sure whether folding should produce a span for `(a,b,c)` or not. Probably yes?

Reply:
> And I'm not sure whether folding should produce a span for (a,b,c) or not

If we are talking about the following example, I think no, because the parameter list is located on the same line 
```ts
(a, b, c) => {
}
```

However, if each parameter is located on a new line, I think it is useful to have folding. 

```ts
(a,
 b,
 c) => { }
```

> Do we already have tests for the multiline parameter list case, like

I couldn't find tests for this case, and it seems that it is not supported, only `{}` handled.

Reply:
I added additional tests. For the following case, _folding_ will be like this

```
[|(a,
 b,
 c) => { 
}|]
```
not 

```
[|(a,
 b,
 c)|] => [|{ 
}|]
```

There is no support _folding_ for parameter list., I'm not sure, maybe it because `) {` can be on the same line and as a result, it will not work correctly. Anyway, it seems better to create a new _feature_ issue and discuss that case there. @sandersn What do you think? 

![1](https://user-images.githubusercontent.com/509265/82563451-c797fd00-9b7f-11ea-8df8-37139378a745.gif)

Reply:
That sounds like a god plan to me. Thanks for the additional analysis.

Reply:
@sandersn thanks.